cmd/clusterctl/cmd/config_repositories_test.go

```diff
@@ -46,6 +47,7 @@ func Test_runGetRepositories(t *testing.T) {
 			g.Expect(err).ToNot(HaveOccurred())
 
 			if val == RepositoriesOutputText {
+				fmt.Print(string(out))
```

Comment:
Leftover?

Reply:
Yeah, left over from testing. Thanks for catching that.